tests/models/rag/test_modeling_rag.py

```diff
@@ -679,6 +680,27 @@ def config_and_inputs(self):
 @require_tokenizers
 @require_torch_non_multi_accelerator
 class RagModelIntegrationTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.temp_dir = tempfile.TemporaryDirectory()
+        cls.dataset_path = cls.temp_dir.name
+        cls.index_path = os.path.join(cls.temp_dir.name, "index")
+
+        ds = load_dataset("hf-internal-testing/wiki_dpr_dummy")["train"]
+        ds.save_to_disk(cls.dataset_path)
+        subprocess.run(
+            [
+                "wget",
+                "-O",
+                f"{cls.index_path}",
+                "https://huggingface.co/datasets/hf-internal-testing/wiki_dpr_dummy/resolve/main/index",
```

Comment:
maybe rename the file "index.faiss" ? and optionally include what kind of index it is, here I assume "flat_index.faiss" ?